### ISSUE TITLE
When I use static_framework import from swift, Archive project can't find umbrella header file.

### ISSUE THREAD

**Issue by noppefoxwolf** [CocoaPods/CocoaPods#7203]
* [x] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

Archive example project of my pod with Xcode.

## What did you expect to happen?

Successfully archive.

## What happened instead?

An error occurred on Xcode.

```
/Users/tomoya.hirano/Library/Developer/Xcode/DerivedData/StaticLib-ezbimhgjbgnycxctfktpybmlxpkt/Build/Intermediates.noindex/ArchiveIntermediates/StaticLib-Example/BuildProductsPath/Release-iphoneos/StaticLib/StaticLib.framework/Modules/module.modulemap:2:19: Umbrella header 'StaticLib-umbrella.h' not found
```

### Stack

```
   CocoaPods : 1.4.0.beta.2
        Ruby : ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-darwin16]
    RubyGems : 2.6.11
        Host : Mac OS X 10.12.6 (16G29)
       Xcode : 9.1 (9B55)
         Git : git version 2.13.6 (Apple Git-96)
Ruby lib dir : /Users/tomoya.hirano/.rbenv/versions/2.4.1/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ b901b19cc1f4a4f428236fdd6cfa290faa03926a
```

### Installation Source

```
Executable Path: /Users/tomoya.hirano/.rbenv/versions/2.4.1/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.1
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.3.0
cocoapods-try         : 1.1.0
```

## Project that demonstrates the issue

https://github.com/noppefoxwolf/StaticLib

### Create Pod library

$ pod lib create StaticLib

What language do you want to use?? [ Swift / ObjC ]
 > ObjC

Would you like to include a demo application with your library? [ Yes / No ]
 >
yes
Which testing frameworks will you use? [ Specta / Kiwi / None ]
 > None

Would you like to do view based testing? [ Yes / No ]
 > No

What is your class prefix?
 > NF

### Add line to `StaticLib.podspec`
`s.static_framework = true`

### Add file to demo application

Class: ViewController
Subclass of: UIViewController
Launguage: Swift

Would you like to configure an Objective-C bridging header?
> create bridging header

### Add file to Pods(StaticLib[StaticLib/Classes])

Class: Dummy
Subclass of: NSObjcet
Launguage: Objective-C

### Add line to ViewController.swift
`import StaticLib`

### pod install
$ cd /Example/
$ pod install

### Archive

Archive with xcode.

```
/Users/tomoya.hirano/Library/Developer/Xcode/DerivedData/StaticLib-ezbimhgjbgnycxctfktpybmlxpkt/Build/Intermediates.noindex/ArchiveIntermediates/StaticLib-Example/BuildProductsPath/Release-iphoneos/StaticLib/StaticLib.framework/Modules/module.modulemap:2:19: Umbrella header 'StaticLib-umbrella.h' not found
```

**Comment by dnkoutso** [comment]
I believe this has been fixed on master. Can you try it?

**Comment by paulb777** [comment]
Verified that master fixes the sample issue here. The fix came in #7164
